CMakeLists.txt

```diff
@@ -26,10 +26,10 @@ add_executable(grblHAL
  tmc_uart.c
  eeprom/eeprom_24AAxxx.c
  eeprom/eeprom_24LC16B.c
- keypad/keypad.c
- sdcard/sdcard.c
- sdcard/ymodem.c
- bluetooth/hc_05.c
+#  keypad/keypad.c
```

Comment:
Please add a comment above explaining why you commented these lines

Reply:
unused dependencies, will remove them instead of commenting